src/test/java/com/google/refine/test/org/deri/reconcile/GRefineServiceManagerTest.java

```diff
@@ -94,7 +95,7 @@ public void saveRdfServiceTest() throws JSONException, IOException{
 		GRefineServiceManager manager = new GRefineServiceManager(registry, dir);
 		
 		Model m = ModelFactory.createDefaultModel();
-		ReconciliationService service = new RdfReconciliationService(id,id, new QueryEndpointImpl(new LarqSparqlQueryFactory(), 
+		ReconciliationService service = new RdfReconciliationService(id,id, new QueryEndpointImpl(new JenaTextSparqlQueryFactory(), 
 				new DumpQueryExecutor(m)), 0);
```

Comment:
trry making changes to the rdf plugin under the extension itself? this test seems to be outside of the extension even though it is part of the rdfplugin